### PR TITLE
PMM-10247 Forced to use install_tarball from its parent directory

### PR DESCRIPTION
**Pull request by cezmunsta**
[PMM-10247](https://jira.percona.com/browse/PMM-10247)

### CODE REVIEW

**Comment by cezmunsta** [comment]
@nikita-b presuming that there is no need for a feature branch for this type of change, let me know if one is needed

**Review by nikita-b — APPROVED** [review]

File: build/bin/install_tarball [hunk 1]
```diff
@@ -73,3 +76,5 @@ if [ "${update}" = "1" ]; then
 else
   ${INSTALL_COMMAND} -m 0660 /dev/null ${PMM_DIR}/config/pmm-agent.yaml
 fi
+
+cd - || true
```
Comment (EvgeniyPatlan) [inline build/bin/install_tarball]: better to save current directory and then just remove to it.
as in future there could be another `cd` command and we would easily return into incorrect directory

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: build/bin/install_tarball (hunk 1 above)
Comment (cezmunsta) [reply build/bin/install_tarball]: @EvgeniyPatlan do you really see the chance of needing another `cd`?

**Review by EvgeniyPatlan — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2613-a67c1c9
client docker - perconalab/pmm-client-fb:PR-2613-a67c1c9
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2613-a67c1c9.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2613-a67c1c9&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2613-a67c1c9.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10578/

**Pull request merged**